code/src/index.css

```diff
@@ -1,13 +1,218 @@
+* {
+  max-width: 100%;
+}
+
 body {
   margin: 0;
   font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
     "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: black;
+  height: 100%
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+/* movie page */
+.movieName {
+  display: flex;
+  position: relative;
+  transition: 0.6;
+  margin-left: 10px;
+  margin-right: 10px;
+}
+
+.movieImage {
+  opacity: 1;
+  border: 2px solid white;
+  margin-top: 15px;
+  margin-left: 10px;
+  margin-right: 10px;
+}
+
+.movieName:hover .movieImage {
+  opacity: 0.3;
+  transition: opacity .3s ease-out;
+  -moz-transition: opacity .3s ease-out;
+  -webkit-transition: opacity .3s ease-out;
+  -o-transition: opacity .3s ease-out;
+}
+
+.movieName:hover .movieText {
+  opacity: 1;
+  transition: opacity .6s ease-out;
+  -moz-transition: opacity .6s ease-out;
+  -webkit-transition: opacity .6s ease-out;
+  -o-transition: opacity .6s ease-out;
+}
+
+.movieText {
+  position: absolute;
+  left: 20px;
+  bottom: 20px;
+  opacity: 0;
+  color: white;
+  transition: 0.6;
+  margin: 10px
+}
+
+.movieTitle {
+  margin-top: 0px;
+  margin-bottom: 10px;
+}
+
+.releaseDate {
+  margin-top: 0px;
+  margin-bottom: 0px;
+  font-size: 16px;
+  font-style: normal;
+}
+
+/* details page  */
+
+.backButton {
+  max-width: 35px;
+  filter: invert(100%);
+  margin-top: 2.5%;
+  margin-left: 2.5%;
+  transition: transform 0.6s;
+}
+
+.backButton:hover {
+  transform: scale(1.2);
+}
+
+.backButton:hover .backToMoviesText {
+  transform: scale(1.2);
+}
+
+.backToMoviesText {
+  margin-top: 2.5%;
+  margin-left: 10px;
+  margin-bottom: 0;
+  font-size: 150%;
+  font-weight: 500;
+}
+
+.goBack {
+  display: block;
+  align-items: center;
+  color: white;
+  text-decoration: none;
+  margin-bottom: 10%;
+}
+
+
+.backWrapper {
+  display: flex;
+  flex-direction: row;
+  transition: transform 0.6s;
+}
+
+.movieContainer {
+  display: grid;
+  grid-template-columns: 1fr;
+  margin-right: 20px;
+}
+
+.productPageContainer {
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 100%;
+  min-height: 100vh;
+}
```

Comment:
Great job with the backdrop poster! We think you can remove some of the styling, because when you use "cover", there's no risk of the image repeating and the width is already 100% (https://www.w3schools.com/cssref/css3_pr_background-size.asp). 

We also found this classname a little unclear - the "products"-part got  us a little bit confused. Maybe change it to something more describing 😊